README.md

```diff
@@ -1,2 +1,29 @@
-# Stellar Account-Viewer 2.0
+# Stellar Account Viewer 2.0
 
+A simple tool to view an account on the Stellar network and make transactions
+from it.
+
```

Comment:
I just thought of this. But maybe a good idea to mention how it's the newer version of the old AVN.

"This app replaces the original Account Viewer [here](https://github.com/stellar/account-viewer) with an updated framework and design."

To explain the "2.0"